.github/workflows/continuous-integration.yaml

```diff
@@ -12,9 +12,9 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Setup Regal
-        uses: StyraInc/setup-regal@v0.2.0
+        uses: StyraInc/setup-regal@v1
         with:
           version: v0.11.0
```

Comment:
https://docs.styra.com/regal/rules/idiomatic/use-if
https://docs.styra.com/regal/rules/idiomatic/use-contains
https://docs.styra.com/regal/rules/idiomatic/boolean-assignment
https://docs.styra.com/regal/rules/imports/use-rego-v1

A lot of the lint warnings in newer regal are related to these findings - I could take a pass at updating some of those separately if there's interest.